src/props.ts

```diff
@@ -1,11 +1,10 @@
 import { Props } from './types'
 
 export const defaultProps: Props = {
-  a11y: true,
```

Comment:
There is no summary for removing this option.

Reply:
I will be writing in migration guide, but basically this reason: https://atomiks.github.io/tippyjs/accessibility/#use-natively-focusable-elements

I don't think it really helps and they should never use a `<span>` in the first place for example.